src/message/messageActionSheet.js

```diff
@@ -74,6 +134,9 @@ export const constructActionButtons = ({ message, auth, narrow, subscriptions, m
   return buttons;
 };
 
-export const executeActionSheetAction = ({ title, ...props }) => {
-  actionSheetButtons.find(x => x.title === title).onPress(props);
+export const executeActionSheetAction = ({ title, ...props }: ExecuteActionSheetActionType) => {
+  const button = actionSheetButtons.find(x => x.title === title);
+  if (button) {
```

Comment:
Is this just for Flow's purpose?
Because I don't think we can have an undefined in practice.